katcp/server.py

```diff
@@ -1055,6 +1056,9 @@ def handle_request(self, connection, msg):
                     def async_reply(f):
                         try:
                             connection.reply(f.result(), msg)
+                            if getattr(handler, 'concurrent_reply', False):
+                                self._logger.debug(
+                                    "%s FUTURE CONCURRENT replied", msg.name)
```

Comment:
Any particular reason to print this only for concurrent-reply futures, rather than for all future replies?